akka-http-core/src/main/scala/akka/http/impl/engine/server/HttpServerBluePrint.scala

```diff
@@ -51,12 +51,15 @@ private[http] object HttpServerBluePrint {
     stack(settings, remoteAddress, log)
 
   def stack(settings: ServerSettings, remoteAddress: Option[InetSocketAddress], log: LoggingAdapter)(implicit mat: Materializer): Http.ServerLayer = {
-    userHandlerGuard(settings.pipeliningLimit) atop
-      requestPreparation(settings, remoteAddress) atop
-      controller(settings, log) atop
-      parsingRendering(settings, log) atop
-      websocketSupport(settings, log) atop
-      unwrapTls
+    val theStack =
+      userHandlerGuard(settings.pipeliningLimit) atop
+        requestPreparation(settings) atop
```

Comment:
Right, and if we want to _really hide it_, we can by making:

```
final case class Attributes(/* this: */attributeList: List[Attributes.Attribute] = Nil) {
```

a `def` instead that hides any internal Attributes. 

That's the only place where currently someone may discover this internal attribute.